src/main/java/com/spotinst/sdkjava/model/bl/ocean/ecs/ClusterLaunchSpecification.java

```diff
@@ -24,6 +24,8 @@
     private List<LaunchSpecTagsSpecification>         tags;
     private String                                    userData;
     private String                                    id;
+    private String                                    keyPair;
```

Comment:
Didn't find these two parameters in the LaunchSpecification.

Reply:
Parameters keyPair and Instance Types are available in JSON when we check 'view configuration' for Cluster on UI. So Added it. 
So As Discussed, Currently aligning with postman and removing those parameters.